triggers.gs

```diff
@@ -0,0 +1,77 @@
+/**
+* Trigger for thr rule for the number of executions for the script
+*/
+function mostExecutedTrigger() {
+  var files = DriveApp.getFilesByName("Admin Rules For Apps Scripts");
+  if (files.hasNext()) {
+    var file = SpreadsheetApp.open(files.next());
+    var active  = file.getSheetByName("Admin Rules"); 
```

Comment:
hard coded text again. save these in a file say, constants.gs
Also write a comment explaining what it is

Reply:
Updated it